.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,19 +3,19 @@ description: Create a bug report to help us address errors in the repository
 title: "[BUG] <bug description>"
 labels: [bug]
 body:
-  - type: input
+  - type: textarea
     id: bugdescription
     attributes:
       label: Description of the bug
     validations:
       required: true
-  - type: input
+  - type: textarea
```

Comment:
Hmm, this should probably be `input` as it is just a one-line text as seen in the example. 🙂

Reply:
I would leave it as textarea for me description of the bug includes more than one line of comment :)

Reply:
This is not for the bug description, it is for the `Project stack or language` part. 🙂